src/packages/@ncigdc/modern_components/DownloadBiospecimenDropdown/DownloadBiospecimenDropdown.relay.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React from 'react';
+import { graphql } from 'react-relay';
+import { compose, withPropsOnChange } from 'recompose';
+import Query from '@ncigdc/modern_components/Query';
+
+export default (Component: ReactClass<*>) =>
+  compose(
+    withPropsOnChange(['filters'], ({ filters }) => {
+      return {
+        variables: {
+          filters,
+        },
+      };
+    }),
+  )((props: Object) => {
+    return (
+      <Query
+        parentProps={props}
+        style={{ width: 'auto' }}
+        variables={props.variables}
+        Component={Component}
+        query={graphql`
+          query DownloadBiospecimenDropdown_relayQuery(
+            $filters: FiltersArgument
```

Comment:
This query doesn't really seem to match the name. You're just trying to get the total number of cases right?